.github/workflows/release.yml

```diff
@@ -1,31 +1,48 @@
-name: release
+name: Release
+
 on:
   release:
     types: [created]
+
 permissions:
   contents: write
+
 jobs:
-  build:
+  release:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v4
         with:
-          go-version: "1.22"
-      - uses: actions/checkout@v2
-      - uses: actions/cache@v2
+          fetch-depth: 0
```

Comment:
qual seria a utilidade de usar `fetch-depth` aqui?

Reply:
Tbm não sei direito pra que serve e não entendi direito na doc, vou perguntar pra debora e retorno.

Reply:
@debora-be